src/components/List/List.jsx

```diff
@@ -11,10 +14,33 @@ const List = () => {
 
   const [value, loading, error] = useCollection(firebasePath);
 
-  const handleCheck = (id, lastDate) => {
+  const handleCheck = (id, lastDate, time, times, lastEstimate) => {
     if (has24HoursPassed(lastDate) === false) {
-      const date = new Date();
-      update(token, id, { lastDate: date });
+      const currentDate = new Date();
+
+      if (times >= 1) {
+        const dayOne = new Date(lastDate.toDate());
+        const result = currentDate - dayOne;
+        var diffDay = Math.round(result / (1000 * 60 * 60 * 24));
+
+        const num =
+          times == 1
```

Comment:
What do you think about using the type-safe equality operator "===" instead of "=="

Reply:
it has been solved. Thanks.